sos/report/plugins/storcli64.py

```diff
@@ -0,0 +1,46 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+import os
+
+
+class StorCLI64(Plugin, IndependentPlugin):
+
+    short_desc = 'LSI MegaRAID devices'
+
+    plugin_name = 'storcli64'
+    profiles = ('system', 'storage', 'hardware',)
+    packages = ('storcli',)
+
+    def setup(self):
+        os.environ['PATH'] += os.pathsep + "/opt/MegaRAID/storcli"
+
+        cmd = 'storcli64'
+        subcmds = [
+            'show ctrlcount J',
+            '/call show AliLog',
+            '/call show all',
+            '/call show events',
+            '/call show termlog',
+            '/call/bbu show all',
+            '/call/cv show all',
+            '/call/dall show J',
+            '/call/eall show all',
+            '/call/eall/sall show all',
+            '/call/sall show all',
+            '/call/vall show all',
+        ]
+
+        logpath = self.get_cmd_output_path()
+
+        self.add_cmd_output([
+            "%s %s" % (cmd, subcmd) for subcmd in subcmds
+        ], runat=logpath)
```

Comment:
What's the rationale behind using `runat` here? It's certainly acceptable to do so if needed, but it's very rare to see so I just would like to understand the requirement here.

Reply:
The command generates a log file `storcli.log` in current directory. If `runat` is not passed, then the log file is generated in the plugin directory or some other.

Reply:
This problem is there with the megacli plugin too. The plugin generates a couple of log files in the current working directory.